src/Console/Command/FixCommand.php

```diff
@@ -297,9 +300,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $resolver->shouldStopOnViolation()
         );
 
+        $this->eventDispatcher->addListener(FixerFileProcessedEvent::NAME, [$progressOutput, 'onFixerFileProcessed']);
```

Comment:
hm, I commented it out and no test failed

Reply:
It was previously inside `ProcessOutput`, I just moved it here to bind events with any progress output (the one that is configured and used). I believe there wasn't any integration test for that, there are only unit tests for progress outputs that don't require event dispatcher, the `onFixerFileProcessed()` method is called directly.

Reply:
ref https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6848#issuecomment-1616171811

i do not understand what decision you look for here. what is the question?

Reply:
If I should add such tests (IMHO out of the scope of this PR) or not 😉.

Reply:
i'm ok to have it non-blocking, can be out of scope, as already missing on master.